sources/main.py

```diff
@@ -64,6 +65,11 @@ async def get_waka_time_stats(repositories: Dict, commit_dates: Dict) -> str:
             os_list = no_activity if len(data["data"]["operating_systems"]) == 0 else make_list(data["data"]["operating_systems"])
             stats += f"💻 {FM.t('operating system')}: \n{os_list}\n\n"
 
+        if EM.SHOW_LAST_7_DAY_CODE_TIME:
+            DBM.i("Adding user last 7 day coding time info...")
+            week_list = no_activity if len(summary_data["data"]) == 0 else make_last_7_day_time_list(summary_data["data"])
+            stats += f"⏳ {'WeekDays'}: \n{week_list}\n\n"
```

Comment:
I didn't modify the `translation.json` file because I was worried that I would mistranslate "Show time spent last 7 days", and I didn't use `FM.t` function here, so I hope someone can help me with the translation.